staging/src/k8s.io/client-go/discovery/cached/memory/memcache.go

```diff
@@ -52,6 +53,7 @@ type memCacheClient struct {
 	groupList              *metav1.APIGroupList
 	cacheValid             bool
 	openapiClient          openapi.Client
+	aggregatedDiscovery    bool
```

Comment:
can I get a comment indicating that this is an indication of whether the cached data is from aggregated discovery and not an indication of desired API to use?